components/configuration-accessor/default/jest.test.mjs

```diff
@@ -21,7 +21,7 @@ const loader_url = "file:///A:/base/lib/node/loader-standalone.mjs";
 // source //
 assertEqual(doesSupportSource("jest --argv", "/bin/sh"), true);
 assertDeepEqual(hookCommandSource("jest --argv", "/bin/sh", base), [
-  `jest --runInBand --setupFilesAfterEnv ${recorder_path.replace(
```

Comment:
I don't think you meant to change THIS instance. :)

Reply:
Right, rebased in b8ff423f4b9d3f22de34d94a7acfb8add4ec4977 Thanks!